drivers/pinctrl/pinctrl_mcux_rt.c

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2021 NXP
```

Comment:
2022 :-)?

Reply:
Thanks for noticing, fixed this and a few other copyrights for new files